src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
You are right that will support both.  And make all the test cases for color animation currently commited still work.  

Then we would just have to figure out how to add `el.components.material.schema.color.type = 'color'` to the generated entity to test the cases of `material.color` or `material.specular`